.travis.yml

```diff
@@ -15,9 +15,9 @@ matrix:
     script:
       - HOMEBREW_NO_AUTO_UPDATE=1 brew install shellcheck
       # Check Parent Dir (No *.sh files curently. Change "*.tool" to "*.{sh,tool}" if any is added later)
-      - shellcheck /home/travis/build/acidanthera/OpenCorePkg/*.tool
+      - shellcheck ./*.tool
```

Comment:
In fact, I think you want `shellcheck ./*.tool || exit 1` here, because otherwise the error may not be reported?

Reply:
It will exit with "1" once there is an error.

Reply:
shellcheck will, but travis does not abort build on a command returning non-zero code.

Reply:
It will continue the build but report that test as failed.
That way can see that Test A failed and Test D failed for amendments.

Reply:
Please confirm.

Do you prefer to abort on the first error met (Your Suggested Change ... Should really then be done for each test. I.e., running macbuild etc should also have similar ) or continue and say so many tests failed (This is how everything is currently setup)?

Reply:
I guess either way is fine to me in this case, I will merge.